Added dustwood recipe with Mortar

Ceredick opened the pull request:
## What
I found it quite annoying that you can’t craft paper in the Stone Age, even though it’s required for arrows in one of the Stone Age quests. Considering paper existed long before steam machines, it makes no sense for paper to be gated behind steam macerator or rarely from a chopping block . Other GregTech packs include it this recipe. 

## Outcome
with the mortar you macerate wood log to one wood dust/pulp

## Additional Information
the output is suppose to be 2 wood dust, but I think it will be better if its kept at 1 per log.

Ceredick committed:
Added dustwood recipe with Mortar

bruberu reviewed (changes requested):
This feels slightly too unrealistic (would you actually smash a log with a mortar and pestle?)

Perhaps a recipe like filing/grinding a stick would be better suited for SUSY.

Ceredick committed:
Changed woodpulp to be crafted with sticks

mortar wood stick to get a small wood pulp
mortar long wood stick to get a wood pulp
